<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PayrollMonths</class>
 <widget class="QWidget" name="PayrollMonths">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>624</width>
    <height>524</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Payroll Months:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTreeView" name="treeView"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Details</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="1" colspan="2">
         <widget class="QLineEdit" name="MonthName"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Month Name:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Year:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QLineEdit" name="Year"/>
        </item>
        <item row="2" column="2">
         <widget class="QCheckBox" name="Closed">
          <property name="text">
           <string>Closed</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Start Date:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" colspan="2">
         <widget class="QDateEdit" name="StartDate">
          <property name="date">
           <date>
            <year>2012</year>
            <month>1</month>
            <day>1</day>
           </date>
          </property>
          <property name="displayFormat">
           <string>dd-MMM-yyyy</string>
          </property>
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>End Date:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="4">
         <widget class="QDateEdit" name="EndDate">
          <property name="date">
           <date>
            <year>2012</year>
            <month>1</month>
            <day>1</day>
           </date>
          </property>
          <property name="displayFormat">
           <string>dd-MMM-yyyy</string>
          </property>
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QToolButton" name="cmdAddNew">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="rc.qrc">
           <normaloff>:/images/Icon_193.ico</normaloff>:/images/Icon_193.ico</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="cmdSave">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="rc.qrc">
           <normaloff>:/images/Icon_10.ico</normaloff>:/images/Icon_10.ico</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="cmdDelete">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="rc.qrc">
           <normaloff>:/images/Icon_136.ico</normaloff>:/images/Icon_136.ico</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="rc.qrc"/>
 </resources>
 <connections/>
</ui>
